base/install_pydevsup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -ex
+
+. /opt/epics/install-functions.sh
+
+download_from_github mdavidsaver pyDevSup $PYDEVSUP_VERSION
+echo PYTHON=python3 >> pyDevSup/configure/CONFIG_SITE
+install_module pyDevSup PYDEVSUP "
+EPICS_BASE
+"
```

Comment:
Is this a separate file because it's not working with Alpine, so couldn't be included in `install_modules.sh`?